### PR TITLE
Feat/django

### PR DESCRIPTION
**Pull request by sooraj1002**
Training API is an unknown #33

### CODE REVIEW

File: workflow/task.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+                response = self.call_llm_generate(user_prompt, workflow_type, llm_model)
+                if response:
+                    cleaned_data = response.strip("`json \n")
+                    parsed_response = json.loads(cleaned_data)
+                    self.validate_json(parsed_response, config.json_schema)
+                    self.parse_and_save_examples(workflow_id, parsed_response)
+                    return parsed_response
+            except Exception as e:
+                print(f"Error generating examples: {str(e)}")
+                # self.generate_or_refine(
```
Comment (KDwevedi) [inline workflow/task.py:101]: Why do we have this section commented out?

File: workflow/views.py [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+@api_view(["POST"])
+def train(request):
+    # TBD
+    return JsonResponse({"message": "hey"})
+
+
+# what is task table
```
Comment (KDwevedi) [inline workflow/views.py:539]: If these comments are relevant, let's create a ticket out of them
If not, let's remove them

File: workflow/views.py [hunk 3]
```diff
@@ -0,0 +1,543 @@
...
+            {"message": "User created successfully!", "user": serializer.data},
+            status=status.HTTP_201_CREATED,
+        )
+    else:
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+@api_view(["POST"])
+def train(request):
+    # TBD
```
Comment (KDwevedi) [inline workflow/views.py:535]: Can we create a ticket out of this?
Why is it TBD?

**Review by KDwevedi — COMMENTED** [review]
Minor cleanups and requests for extracting out contextual information from code comments to repo issues.

Otherwise LGTM

File: workflow/views.py (hunk 3 above)
Comment (sooraj1002) [reply workflow/views.py:535]: This is the final step after generating the samples. We had decided to work on this once the first part is completed

Comment (sooraj1002) [reply workflow/views.py:535]: https://github.com/BharatSahAIyak/autotune/issues/33

File: workflow/views.py (hunk 2 above)
Comment (sooraj1002) [reply workflow/views.py]: removed https://github.com/BharatSahAIyak/autotune/commit/8553a17a998c9768bbe87d3c39f3ceeb2c363a53

**Review by KDwevedi — CHANGES_REQUESTED** [review]
- [ ] Note what expected functionality of Autotune is limited by #33
  - [ ] Add a desc of this limited functionality to the [pr comment](https://github.com/BharatSahAIyak/autotune/pull/32#issue-2254715351) 
- [ ] [Make the service ready for deployment](https://github.com/BharatSahAIyak/docker-bhasai/blob/dev/docs/onboarding.md)

**Pull request merged**